### PR TITLE
Create BookAugmentedStatusTest.java

### PR DESCRIPTION
**Pull request by naviza**
Test cases for the augmented book status based on context. These tests show the basic flow of a book exchange starting from when the potential borrower looks at the book to the borrower returning the book to the owner.

### CODE REVIEW

**Review by n00b4ever — APPROVED** [review]

File: app/src/test/java/com/example/bookwormadventuresdeluxe2/BookAugmentedStatusTest.java [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+
+    /**
+     * User requested the book
+     */
+    @Test
+    public void testGetAugmentStatusRequested()
+    {
+        book.addRequester(testBorrower);
+        book.setStatus(Status.Available);
+        assertEquals(book.getAugmentStatus(testBorrower), Status.Requested);
```
Comment (JawadHossain) [inline app/src/test/java/com/example/bookwormadventuresdeluxe2/BookAugmentedStatusTest.java:61]: assertEquals(book.getAugmentStatus("anotherUser"), Status.Available);
we could add this if not tested anywhere else

File: app/src/test/java/com/example/bookwormadventuresdeluxe2/BookAugmentedStatusTest.java [hunk 2]
```diff
@@ -0,0 +1,136 @@
...
+    /**
+     * The book is being passed from owner to borrower
+     * The borrower is viewing the book
+     */
+    @Test
+    public void testGetAugmentStatusbPendingBorrower()
+    {
+        book.addRequester(testBorrower);
+        book.setStatus(Status.bPending);
+        assertEquals(book.getAugmentStatus(testBorrower), Status.Accepted);
```
Comment (JawadHossain) [inline app/src/test/java/com/example/bookwormadventuresdeluxe2/BookAugmentedStatusTest.java:96]: also maybe assertEquals(book.getAugmentStatus("owner"), Status.Borrowed); ?

Comment (naviza) [reply app/src/test/java/com/example/bookwormadventuresdeluxe2/BookAugmentedStatusTest.java:96]: I have that in the test above but I'll probably combine the two.

File: app/src/test/java/com/example/bookwormadventuresdeluxe2/BookAugmentedStatusTest.java (hunk 1 above)
Comment (naviza) [reply app/src/test/java/com/example/bookwormadventuresdeluxe2/BookAugmentedStatusTest.java:61]: Good call.

**Review by JawadHossain — APPROVED** [review]

**Pull request merged**